### PR TITLE
Rename left out decisions to commands, workflow task

### PR DESCRIPTION
**Pull request by feedmeapples**
## What does this PR do?
Renames "decision" terminology to Workflow task/Command

### CODE REVIEW

File: docs/go/retries.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ For a Workflow with `RetryPolicy`:
 
 - If a Workflow fails and a retry policy is configured for it, the Workflow execution will be closed with a `ContinueAsNew` event.
   This event will have the `ContinueAsNewInitiator` field set to `RetryPolicy` and the new `RunId` for the next retry attempt.
-- The new attempt will be created immediately. But the first decision task won't be scheduled until the backoff duration. That duration is recorded as the `firstDecisionTaskBackoffSeconds` field of the new run's `WorkflowExecutionStartedEventAttributes` event.
+- The new attempt will be created immediately. But the first workflow task won't be scheduled until the backoff duration. That duration is recorded as the `FirstWorkflowTaskBackoff` field of the new run's `WorkflowExecutionStartedEventAttributes` event.
```
Comment (feedmeapples) [inline docs/go/retries.md:96]:
```suggestion
- The new attempt will be created immediately. But the first workflow task won't be scheduled until the backoff duration. That duration is recorded as the `firstWorkflowTaskBackoff` field of the new run's `WorkflowExecutionStartedEventAttributes` event.
```

**Review by flossypurse — APPROVED** [review]

File: docs/server/security.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ The following arguments must be passed to the `Authorize` method for example:
 <!--SNIPSTART temporal-common-authorization-authorizer-calltarget-->
 <!--SNIPEND-->
 
-The `Authorize` method then returns one of two possible decisions within the `Result.Decision` field:
```
Comment (alexshtin) [inline docs/server/security.md:76]: This is another decision. Please revert.

File: docs/system-tools/tctl.md [hunk 3]
```diff
@@ -359,13 +359,13 @@ tctl workflow reset -w <wid> -r <rid> --reset_type <reset_type> --reason "some_r
 If you are familiar with the Temporal history event, you can also reset to any command finish event by using:
 
 ```bash
-tctl workflow reset -w <wid> -r <rid> --event_id <decision_finish_event_id> --reason "some_reason"
+tctl workflow reset -w <wid> -r <rid> --event_id <workflow_finish_event_id> --reason "some_reason"
```
Comment (alexshtin) [inline docs/system-tools/tctl.md:362]: I think this should be:
```suggestion
tctl workflow reset -w <wid> -r <rid> --event_id <workflow_task_finish_event_id> --reason "some_reason"
```

File: docs/go/versioning.md [hunk 4]
```diff
@@ -149,8 +149,8 @@ type.
 ## Sanity checking
 
 The Temporal client SDK performs a sanity check to help prevent obvious incompatible changes.
-The sanity check verifies whether a decision made in replay matches the event recorded in history,
-in the same order. The decision is generated by calling any of the following methods:
+The sanity check verifies whether a command made in replay matches the event recorded in history,
```
Comment (feedmeapples) [inline docs/go/versioning.md:152]:
```suggestion
The sanity check verifies whether a Command made in replay matches the event recorded in history,
```

File: docs/go/versioning.md [hunk 5]
```diff
@@ -149,8 +149,8 @@ type.
...
+in the same order. The command is generated by calling any of the following methods:
```
Comment (feedmeapples) [inline docs/go/versioning.md:153]:
```suggestion
in the same order. The Command is generated by calling any of the following methods:
```

File: docs/php/versioning.md [hunk 6]
```diff
@@ -146,8 +146,8 @@ type.
 ## Sanity checking
 
 The Temporal client SDK performs a sanity check to help prevent obvious incompatible changes.
-The sanity check verifies whether a decision made in replay matches the event recorded in history,
-in the same order. The decision is generated by calling any of the following methods:
+The sanity check verifies whether a command made in replay matches the event recorded in history,
```
Comment (feedmeapples) [inline docs/php/versioning.md:149]:
```suggestion
The sanity check verifies whether a Command made in replay matches the event recorded in history,
```

File: docs/php/versioning.md [hunk 7]
```diff
@@ -146,8 +146,8 @@ type.
...
+in the same order. The command is generated by calling any of the following methods:
```
Comment (feedmeapples) [inline docs/php/versioning.md:150]:
```suggestion
in the same order. The Command is generated by calling any of the following methods:
```

File: docs/server/security.md [hunk 8]
```diff
@@ -73,7 +73,7 @@ The following arguments must be passed to the `Authorize` method for example:
...
+The `Authorize` method then returns one of two possible commands within the `Result.Decision` field:
```
Comment (feedmeapples) [inline docs/server/security.md:76]:
```suggestion
The `Authorize` method then returns one of two possible decisions within the `Result.Decision` field:
```

File: docs/system-tools/tctl.md [hunk 9]
```diff
@@ -359,13 +359,13 @@ tctl workflow reset -w <wid> -r <rid> --reset_type <reset_type> --reason "some_r
...
 ```bash
-tctl workflow reset -w <wid> -r <rid> --event_id <decision_finish_event_id> --reason "some_reason"
+tctl workflow reset -w <wid> -r <rid> --event_id <workflow_task_finish_event_id> --reason "some_reason"
 ```
 
 Some things to note:
 
 - When reset, a new run will be kicked off with the same workflowId. But if there is a running execution for the workflow(workflowId), the current run will be terminated.
-- decision_finish_event_id is the Id of events of the type: DecisionTaskComplete/DecisionTaskFailed/DecisionTaskTimeout.
+- workflow_finish_event_id is the Id of events of the type: WorkflowTaskComplete/WorkflowTaskFailed/WorkflowTaskTimeout.
```
Comment (feedmeapples) [inline docs/system-tools/tctl.md:368]:
```suggestion
- workflow_task_finish_event_id is the Id of events of the type: WorkflowTaskComplete/WorkflowTaskFailed/WorkflowTaskTimeout.
```

File: docs/server/security.md (hunk 2 above)
Comment (feedmeapples) [reply docs/server/security.md]: good catch

**Pull request merged**